src/ossia-max/src/utils.hpp

```diff
@@ -21,19 +21,50 @@ namespace ossia
 namespace max
 {
 
+void object_namespace(object_base* x);
+
+/**
```

Comment:
thanks for the comments!

Reply:
there are still a lot to do... but it's a start !